### PR TITLE
Arc new tab

### PR DESCRIPTION
**Pull request by thomaspaulmann**
## Description

As mentioned in #4255, I added a preference to define the URL that opens with the "Open New Tab" command. By default it's https://google.com.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/arc/package.json [hunk 1]
```diff
@@ -58,19 +58,31 @@
...
+          "required": false,
+          "default": "https://google.com"
+        }
+      ]
     }
   ],
   "dependencies": {
     "@raycast/api": "^1.45.0",
     "@raycast/utils": "^1.4.0",
+    "is-url": "^1.2.4",
```
Comment (snelling-a) [inline extensions/arc/package.json:77]: do you think it's necessary to use a library for this? it seems like a fairly simple [regex](https://www.oreilly.com/library/view/regular-expressions-cookbook/9781449327453/ch08s01.html) would do the trick.

**Review by snelling-a — APPROVED** [review]
🚀 tested and LGTM. 

One comment about validation of the url. Also, related to that, is it possible to do the validation when the user enters the url in the preferences? Might make for a better UX if the feedback is given there instead of when they try to open a new tab.

Screen Shot 2023-01-16 at 9 59 04 PM

File: extensions/arc/package.json (hunk 1 above)
Comment (thomaspaulmann) [reply extensions/arc/package.json:77]: I checked the package and it does pretty much that, this is why I pulled it in but good research.

**Comment by thomaspaulmann** [comment]
> 🚀 tested and LGTM.
> 
> One comment about validation of the url. Also, related to that, is it possible to do the validation when the user enters the url in the preferences? Might make for a better UX if the feedback is given there instead of when they try to open a new tab.
> 
> Screen Shot 2023-01-16 at 9 59 04 PM

Nice, thanks for testing and verifying it works for you. Unfortunately we don't have validation in the preferences yet. This could be indeed a nice-to-have that would improve the UX. We have plans to revamp the preferences and make it more dynamic and configurable. This should also cover validation. In the meantime I added some validation in the command as you've seen.

**Pull request merged**